Can't build the kernel module for 5.8.1-arch1-1

**Describe the bug**

Building the kernel module for `5.8.1-arch1-1` does not work with `falco-driver-loader` nor with [driverkit](https://github.com/falcosecurity/driverkit).


**How to reproduce it**

- Install Falco 0.24.0 (I have used the binary package)
- run `sudo falco-driver-loader`


For driverkit, follow the instruction for vanilla target describe [here](https://github.com/falcosecurity/driverkit#vanilla).


**Expected behaviour**

`falco-driver-loader` should be able to build the kernel model on Arch (it worked for `5.7.x`)

**Screenshots**

With `falco-driver-loader`:
```
* Running falco-driver-loader with: driver=module, compile=yes, download=yes
* Unloading falco module, if present
* Trying to dkms install falco module

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...
make -j8 KERNELRELEASE=5.8.1-arch1-1 -C /usr/lib/modules/5.8.1-arch1-1/build M=/var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build...(bad exit status: 2)
* Running dkms build failed, dumping /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/make.log
DKMS make.log for falco-85c88952b018fdbce2464222c3303229f5bfcfad for kernel 5.8.1-arch1-1 (x86_64)
Mon 17 Aug 2020 02:50:41 PM CEST
make: Entering directory '/usr/lib/modules/5.8.1-arch1-1/build'
  AR      /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/built-in.a
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/main.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/dynamic_params_table.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/fillers_table.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/flags_table.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_events.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/event_table.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_fillers.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/syscall_table.o
  CC [M]  /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_cputime.o
In file included from ./include/linux/export.h:43,
                 from ./include/linux/linkage.h:7,
                 from ./arch/x86/include/asm/cache.h:5,
                 from ./include/linux/cache.h:6,
                 from ./include/linux/time.h:5,
                 from ./include/linux/compat.h:10,
                 from /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_fillers.c:12:
/var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_fillers.c: In function ‘ppm_get_tty’:
/var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_fillers.c:741:15: error: implicit declaration of function ‘probe_kernel_read’; did you mean ‘kernel_read’? [-Werror=implicit-function-declaration]
  741 |  if (unlikely(probe_kernel_read(&tty, &sig->tty, sizeof(tty))))
      |               ^~~~~~~~~~~~~~~~~
./include/linux/compiler.h:78:42: note: in definition of macro ‘unlikely’
   78 | # define unlikely(x) __builtin_expect(!!(x), 0)
      |                                          ^
cc1: some warnings being treated as errors
make[1]: *** [scripts/Makefile.build:280: /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build/ppm_fillers.o] Error 1
make: *** [Makefile:1756: /var/lib/dkms/falco/85c88952b018fdbce2464222c3303229f5bfcfad/build] Error 2
make: Leaving directory '/usr/lib/modules/5.8.1-arch1-1/build'
```

With `driverkit`:
```
...
DEBU + make KERNELDIR=/tmp/kernel                 
DEBU *make -C /tmp/kernel M=/tmp/driver modules 
DEBU *make[1]: Entering directory '/tmp/kernel' 
DEBU   CC [M]  /tmp/driver/main.o         
DEBU -  CC [M]  /tmp/driver/dynamic_params_table.o 
DEBU &  CC [M]  /tmp/driver/fillers_table.o 
DEBU $  CC [M]  /tmp/driver/flags_table.o  
DEBU #  CC [M]  /tmp/driver/ppm_events.o   
DEBU $  CC [M]  /tmp/driver/ppm_fillers.o  
DEBU HIn file included from ./include/linux/export.h:43, 
DEBU                  from ./include/linux/linkage.h:7, 
DEBU                  from ./arch/x86/include/asm/cache.h:5, 
DEBU                  from ./include/linux/cache.h:6, 
DEBU                  from ./include/linux/time.h:5, 
DEBU                  from ./include/linux/compat.h:10, 
DEBU                  from /tmp/driver/ppm_fillers.c:12: 
DEBU /tmp/driver/ppm_fillers.c: In function 'ppm_get_tty': 
DEBU /tmp/driver/ppm_fillers.c:741:15: error: implicit declaration of function 'probe_kernel_read'; did you mean 'kernel_read'? [-Werror=implicit-function-declaration] 
DEBU   if (unlikely(probe_kernel_read(&tty, &sig->tty, sizeof(tty)))) 
DEBU                ^~~~~~~~~~~~~~~~~             
DEBU ./include/linux/compiler.h:78:42: note: in definition of macro 'unlikely' 
DEBU  # define unlikely(x) __builtin_expect(!!(x), 0) 
DEBU                                           ^  
DEBU +cc1: some warnings being treated as errors 
DEBU Mmake[2]: *** [scripts/Makefile.build:281: /tmp/driver/ppm_fillers.o] Error 1 
DEBU 2make[1]: *** [Makefile:1756: /tmp/driver] Error 2 
DEBU )make[1]: Leaving directory '/tmp/kernel' 
DEBU $make: *** [Makefile:7: all] Error 2  
...
```



**Environment**

<!-- Please complete the following info. -->

- Falco version:
```
Falco version: 0.24.0
Driver version: 85c88952b018fdbce2464222c3303229f5bfcfad
```
- System info:
<!-- Falco has a built-in support command you can use  "falco --support | jq .system_info" -->
- Cloud provider or hardware configuration:
- OS:
`Arch Linux`
- Kernel:
`Linux x1 5.8.1-arch1-1 #1 SMP PREEMPT Wed, 12 Aug 2020 18:50:43 +0000 x86_64 GNU/Linux`
- Installation method:
Binary package

**Additional context**

<!-- Add any other context about the problem here. -->

--- a/cmake/modules/sysdig.cmake
+++ b/cmake/modules/sysdig.cmake
@@ -26,8 +26,8 @@ file(MAKE_DIRECTORY ${SYSDIG_CMAKE_WORKING_DIR})
 # To update sysdig version for the next release, change the default below
 # In case you want to test against another sysdig version just pass the variable - ie., `cmake -DSYSDIG_VERSION=dev ..`
 if(NOT SYSDIG_VERSION)
-  set(SYSDIG_VERSION "85c88952b018fdbce2464222c3303229f5bfcfad")
-  set(SYSDIG_CHECKSUM "SHA256=6c3f5f2d699c9540e281f50cbc5cb6b580f0fc689798bc65d4a77f57f932a71c")
+  set(SYSDIG_VERSION "113c22c4693fac5cc2db1b115c2f1ee0b6b701b7")
+  set(SYSDIG_CHECKSUM "SHA256=48fb9ec586ec217d8509e4d20a94e911a9679c78ddc23877d07c6dda26ff34db")
 endif()
 set(PROBE_VERSION "${SYSDIG_VERSION}")
 

